Add Akka and MLRun files (#227)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/mlrun.md

---
name: MLRun
category: AI/ML
description: MLRun by Iguazio is an open-soure platform that enables developers to quickly build and manage continuous ML applications throughout their lifecycles.
download_url: https://github.com/mlrun/mlrun/releases
works_on_arm: true
supported_minimum_version:
    version_number: 0.1.6
    release_date: 2019/08/28

optional_info:
    homepage_url: https://www.iguazio.com/open-source/mlrun/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content:
        official_docs: https://docs.mlrun.org/en/latest/install.html#install-setup-guide
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Linux/ARM64 release notes are not available. Installation and Testing are done using "pip3 install mlrun".

Author: ```suggestion
    other_info: Linux/ARM64 release notes are not available. Installation is verified using "pip3 install mlrun".
```
```suggestion
    other_info: Linux/ARM64 release notes are not available. Installation and Testing are done using "pip3 install mlrun".
```
